docs/confluence/advanced/grounding.md

```diff
@@ -0,0 +1,154 @@
+# Grounding 
+
+## Flag
+
+Grounding is on by default. To turn it off, use `--settings -smt_groundQuantifiers=false`
+
+ 
+## Grounding General Idea
+
+Grounding should make using quantifiers much easier on the solvers. For example,
```

Comment:
Grounding should make it much easier to use quantifiers on the solvers. 
Example: